lib/webspicy/tester.rb

```diff
@@ -111,7 +111,6 @@ def run_specification
         @service = service
         reporter.before_service
         run_service
-        reporter.service_done
```

Comment:
Nope. If the reporter has something to do when the service is done, we can't just remove the line.

We need to improve the reporter itself and/or change the protocol.

Reply:
@blambeau even if it just doing an `io.puts` ?